changelog.d/+integration-dns.internal.md

```diff
@@ -0,0 +1 @@
+Add integration test for DNS resolution
```

Comment:
This doesn't solve what we wanted to have this for - checking `getaddrinfo` return value. I suggest implementing the test in rust, and asserting in the rust app that domain `X` would return result and domain `Y` would return the appropriate error. 
Having said that, I do see a value in the case shown here (where it's resolved over UDP using `socket/connect/send/recv`) but you'd need to add assertions to the app and also I believe that because the dns queries are being generated "on the fly", each would have different query id, so a constant response wouldn't work.

Reply:
sure, will do, makes sense. was trying to make something basic work in node and then go ahead with implementing other scenarios, ill remove the node test for now and move to rust